inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -283,6 +283,8 @@ public function save_post( $post_id ) {
 	 */
 	public function is_valid_post_type( $post_type ) {
 
+		global $wp_post_types;
```

Comment:
If it gets requested a lot of times, would a static cache of the output of the API function not be better?
The result of public post types would be expected to remain the same during the entire execution (at least that's the hope).